### PR TITLE
Send email after CI finish

### PR DESCRIPTION
**Pull request by JRPan**
Send an email after CI finishes.

### CODE REVIEW

File: test
Comment (barnes88) [inline test:1]: Is there a reason for committing this bin?

File: test.cu [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+{
+  unsigned n = 1024*1024;
+  int results[n];
+
+  int *d_results;
+  cudaMalloc(&d_results, n * sizeof(int));
+
+  int *start, *d_start;
+  cudaMalloc(&d_start, n * sizeof(int));
+  cudaMallocHost(&start, n * sizeof(int));
```
Comment (barnes88) [inline test.cu]: start and d_start are allocated but not freed

File: test
Comment (JRPan) [reply test:1]: absolutely not. lol. removed

File: test.cu (hunk 1 above)
Comment (JRPan) [reply test.cu]: removed

**Comment by JRPan** [comment]
Also I squashed everything. Cleaner.

**Comment by JRPan** [comment]
@barnes88 can we approve this so I can enable the CI?

**Review by barnes88 — APPROVED** [review]
looks fine

**Pull request merged**